lib/Dialect/Comb/CombOps.cpp

```diff
@@ -45,13 +45,15 @@ Value comb::createOrFoldSExt(Value value, Type destTy,
   return createOrFoldSExt(builder.getLoc(), value, destTy, builder);
 }
 
-Value comb::createOrFoldNot(Location loc, Value value, OpBuilder &builder) {
+Value comb::createOrFoldNot(Location loc, Value value, OpBuilder &builder,
+                            bool twoState) {
   auto allOnes = builder.create<hw::ConstantOp>(loc, value.getType(), -1);
-  return builder.createOrFold<XorOp>(loc, value, allOnes, false);
+  return builder.createOrFold<XorOp>(loc, value, allOnes, twoState);
```

Comment:
It saddens me that I had to check the Verilog spec to verify not and xor had the same truth table for 'x and 'z.  (Tables 11-13 and 11-15).